### DISCUSSION TITLE
How to add .env in to .gitignore

### DISCUSSION THREAD

**Discussion by rgbskills** [codesandbox/codesandbox-client#4875] [General] [answered]
I tried to add .env to the .gitignore file but it wont ignore it. How do I make it so its ignored ?

**Comment by garethx** [comment, marked as answer]
We don't support that as we don't yet have proper git integration (it's on the roadmap). We have our own feature for secrets in container sandboxes (https://codesandbox.io/docs/secrets), so if you're able to then we suggest using that instead.

**Comment by rgbskills** [comment]
Thank you for the quick reply, I can use the secrets feature but I need .env on the server anyway so keep me updated :)
